### PR TITLE
Webpack first step

### PR DESCRIPTION
**Pull request by RHK-MICROVERSE**
Webpack Server along with distribution in development server deployment

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
checked!

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Rassel, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 

## Highlights 👍  
🎲  Webpack setup correctly!
🎲 App runs correctly!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- I think your ReadMe intro section has rather too much content. It will therefore be a good idea if you streamline this section so as to make it less bulky for visitors.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,267 @@
...
+<!-- > Describe your project in 1 or 2 sentences. -->
+### How to build the "To Do List" 
+- 1: Manage Daily task (plain JS with objects).
+- 2: Shows list of tasks to be done (plain JS with classes) and add basic CSS.
+- 3: Create a complete List of tasks to be done and update it.
+
+Will start by building the core functionalities and ignoring how the website looks. At the very beginning the website will look similar to the UI below. Note that it is plain HTML with no styling, but it will allow to add and remove books from the list!
+
+<p align="center">
+  <img src="./images/first_apperance.png" alt="Basic UI" width="300px" />
```
[the remark this answers is not in the document]
Comment (Rassel-AI) [reply README.md:51]: It is done. sorry for the misktake in the file name.

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**